### ISSUE TITLE
Other languges implementation?

### ISSUE THREAD

**Issue by NK990** [PlayVoice/vits_chinese#55]
Is it possible to implement other languges?

**Comment by NikitaKononov** [comment]
> Is it possible to implement other languges?

yes, vits eats phonemes, so you just need to change the text specific part

**Comment by NikitaKononov** [comment]
> Is it possible to implement other languges?

sorry, I was wrong
you also need to use bert for your language

**Comment by hwhw97** [comment]
> > Is it possible to implement other languges?
> 
> yes, vits eats phonemes, so you just need to change the text specific part

请问有更多的教程吗？谢谢~
